### PR TITLE
Fixes #5565: Revise EPUB renderer z-indices

### LINKED ISSUE #5565: EPUB: components and loading error show above floating app top bar
### Observed behavior

observed by @bjester:

> image

> 
image

### Expected behavior

Z-indexes should be respected

### User-facing consequences

slightly broken-looking behavior

### Steps to reproduce

* make window small enough that the app bar will scroll offscreen
* scroll down and then back up again

### Context

Linux, Chrome, 0.12.4-beta4

### PR DESCRIPTION
**Pull request by bjester**
### Summary

- Updated all z-index values to be within 0 and 4
epub-zindex

### Reviewer guidance

- Open an EPUB
- Decrease page height until app top bar disappears on scroll
- Verify no EPUB renderer components appear above (Z axis) the app top bar
- Verify EPUB toolbar buttons properly toggle sidebars

### References

https://github.com/learningequality/kolibri/issues/5565

----

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] ~If this is an important user-facing change, PR or related issue has a 'changelog' label~
- [ ] ~If this includes an internal dependency change, a link to the diff is provided~

Testing:

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] ~Critical user journeys are covered by Gherkin stories~
- [ ] ~Critical and brittle code paths are covered by unit tests~

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

**Comment by indirectlylit** [comment]
This may be out of scope for this issue, so let me know if you'd like me to file a new one.

One thing to watch for is that the drop shadows are kept consistent with elevation. We have a set of [standard drop shadows](https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/styles/materialDesignElevations.scss) that we're not using here.

It's also not clear to me why the header and footer elements have drop shadows or elevations at all.  This is not consistent with Material [elevation conventions](https://material.io/design/environment/elevation.html) or our other renderers. (cc @khangmach @jtamiace)

| current | flattened |
|--|--|
| image | image  |

**Comment by bjester** [comment]
@indirectlylit Let's file a new issue for the drop shadows.

**Comment by indirectlylit** [comment]
cool. I've expanded an existing issue to encompass a few more UI inconsistencies: https://github.com/learningequality/kolibri/issues/5564

File: kolibri/plugins/document_epub_render/assets/src/views/BottomBar.vue [hunk 1]
```diff
@@ -96,7 +96,7 @@
   @import './EpubStyles';
 
   .bottom-bar {
-    z-index: 8;
+    z-index: 0;
```
Comment (indirectlylit) [inline kolibri/plugins/document_epub_render/assets/src/views/BottomBar.vue]: can this whole line be deleted?

**Review by indirectlylit — COMMENTED** [review]
code looks fine. @jonboiser would you mind testing & merging

File: kolibri/plugins/document_epub_render/assets/src/views/BottomBar.vue (hunk 1 above)
Comment (bjester) [reply kolibri/plugins/document_epub_render/assets/src/views/BottomBar.vue]: Looks like it.

**Review by jonboiser — APPROVED** [review]
Looks good to me!

**Pull request merged**